packages/contracts/src/BaseRelayRecipient.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-3.0-only
 // solhint-disable no-inline-assembly
-pragma solidity >=0.7.6;
+pragma solidity ^0.8.0;
```

Comment:
interfaces (and base classes) are better set using `>=`, so that they can be used with later versions.
If they don't change, can even leave them as `>= 0.7.6`, so that clients with earlier version can work.

Reply:
I agree that it would be good if we keep them as `>=0.7.6` for the contracts if possible just so at least it works with the earlier version. Some projects may still rely on an earlier version.